src/main/java/com/softserve/academy/spaced/repetition/service/AccountService.java

```diff
@@ -31,4 +31,17 @@ public void updateLearningRegime(LearningRegime learningRegime) throws NotAuthor
         account.setLearningRegime(learningRegime);
         accountRepository.save(account);
     }
+
+    public int getCardsNumber() throws NotAuthorisedUserException {
+        return accountRepository.findOne(userService.getAuthorizedUser().getAccount().getId()).getCardsNumber();
```

Comment:
Why can't you just write
`return userService.getAuthorizedUser().getAccount().getCardsNumber();`
without checking the database each time?